src/lib/db/access-store.ts

```diff
@@ -144,7 +144,6 @@ export class AccessStore implements IAccessStore {
                 .join(`${T.GROUP_ROLE} AS gr`, 'gu.group_id', 'gr.group_id')
                 .join(`${T.ROLE_PERMISSION} AS rp`, 'rp.role_id', 'gr.role_id')
                 .join(`${T.PERMISSIONS} AS p`, 'p.id', 'rp.permission_id')
-                .whereNull('g.root_role_id')
```

Comment:
Why are we deleting this line?

Reply:
Because otherwise, we won't be able to find project roles for groups with root roles.